server/api/src/rsGeneralAdmin.cpp

```diff
@@ -1026,6 +1027,20 @@ _rsGeneralAdmin( rsComm_t *rsComm, generalAdminInp_t *generalAdminInp ) {
             rei.uoip = &rsComm->proxyUser;
             args[0] = generalAdminInp->arg2;
             args[1] = generalAdminInp->arg3;
+
+            // Verify the zone name satisfies the requirements defined by the configuration
+            // schema. It would be nice to get the regular expression from the schema file, but
+            // we can't do that because iRODS 4 supports remote schema files. However, hard-coding
+            // the regular expression is also fine because no one should modify the schema (even
+            // though it's modifiable).
```

Comment:
I have the same comment on this comment in my previous comment.

If the comment is to highlight potential future issues by hard-coding the regex, the comment should be simplified to that. State the assumption of a default iRODS 4 regex and that issues may arise if the regex file is changed (if I understood the comment).

Reply:
Removed comment.